packages/integrations/cloudflare/README.md

```diff
@@ -191,6 +191,42 @@ export default defineConfig({
 });
 ```
 
+## WASM module imports
+
+`wasmModuleImports: boolean`
+
+Cloudflare has native support for importing `.wasm` files [directly as ES modules](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration). To enable importing them as modules in the cloudflare build and astro dev server, add `wasmModuleImports: true` to your config.
```

Comment:
```suggestion
default: `false`

Whether or not to import `.wasm` files [directly as ES modules](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration). 

Add `wasmModuleImports: true` to `astro.config.mjs` to enable in both the Cloudflare build and the Astro dev server. 
```
This seems more in keeping with the option directly above, stating the default and firstly, defining the configuration setting itself.